README.md

```diff
@@ -7,32 +7,136 @@ Inspired from [Caddy](https://github.com/mholt/caddy).
 ## Features
 
 * Easy configuration with Maddyfile (same as Caddyfile)
-* Automatic TLS and DKIM support
 * Transparent PGP support
 * Runs anywhere with no external dependencies (not even libc)
 
-## Usage
+## Installation
 
 ```shell
-go get github.com/emersion/maddy/...
+go get github.com/emersion/maddy/cmd/maddy
 ```
 
-Create a file named `Maddyfile`:
+Build tags:
+* `nosqlite3`
+  Disable SQLite3 support in go-sqlmail (enabled by default). Saves around 9
+  MiB of binary size.
+* `mysql`
+  Include support for MySQL driver in go-sqlmail.
+* `postgresql`
+  Include support for PostgreSQL driver in go-sqlmail.
 
+## Configuration
+
+#### Syntax
```

Comment:
This is using one heading level more than necessary

Reply:
Ok. Will fix.